Libraries/Image/Image.ios.js

```diff
@@ -88,6 +90,33 @@ var Image = React.createClass({
      * testing scripts.
      */
     testID: PropTypes.string,
+    /**
+     * Event callback function to invoke when image load begins.
+     * Includes the following event arguments:
+     *
+     * - target: Image component instance,
+     * - uri: Network image source
+     */
+    onLoadingStart: PropTypes.func,
```

Comment:
I agree with @philikon, it would be nice to follow web naming convention here.

- onLoadStart: http://www.w3schools.com/jsref/event_onloadstart.asp
- onLoad: http://www.w3schools.com/jsref/event_onload.asp
- onError: http://www.w3schools.com/jsref/event_onerror.asp